src/ApiTestCase.php

```diff
@@ -145,7 +167,7 @@ protected function get($id)
      */
     protected function assertResponseCode(Response $response, $statusCode)
     {
-        $this->assertEquals($statusCode, $response->getStatusCode(), $response->getContent());
+        static::assertEquals($statusCode, $response->getStatusCode(), $response->getContent());
```

Comment:
What is a benefit of these changes? According to documentation we should use the [previous form](https://phpunit.de/manual/current/en/appendixes.assertions.html)